spec/services/encryption/kms_logger_spec.rb

```diff
@@ -4,38 +4,39 @@
   describe '.log' do
     context 'with a context' do
       it 'logs the context' do
-        described_class.log(:encrypt, context: 'pii-encryption', user_uuid: '1234-abc')
+        log = {
+          kms: {
+            action: 'encrypt',
+            encryption_context: { context: 'pii-encryption', user_uuid: '1234-abc' },
+          },
+        }.to_json
 
-        log_contents = File.read('log/kms.log').split("\n")
-        log = log_contents.last
+        expect(described_class.logger).to receive(:info).with(log)
```

Comment:
Where is `:info` defined?

Reply:
It is part of ruby's standard lib. It is defined here: https://github.com/ruby/ruby/blob/c92c0a593593da2eb1ff94d83d80f71e7ae5343c/lib/logger.rb#L524

Reply:
Ohh.. for some reason I was thinking of something other than the logger.  Thanks!